packages/api/src/graphql/blocks.ts

```diff
@@ -702,6 +768,21 @@ export const GraphQLTeaserGridBlockInput = new GraphQLInputObjectType({
   }
 })
 
+export const GraphQLFlexTeaserInput = new GraphQLInputObjectType({
+  name: 'FlexTeaserInput',
+  fields: {
+    teaser: {type: GraphQLTeaserInput},
+    alignment: {type: GraphQLNonNull(GraphQLFlexGridItemAlignmentInput)}
+  }
+})
+
+export const GraphQLTeaserFlexGridBlockInput = new GraphQLInputObjectType({
+  name: 'TeaserFlexGridBlockInput',
+  fields: {
+    flexTeasers: {type: GraphQLNonNull(GraphQLList(GraphQLFlexTeaserInput))}
```

Comment:
@roosnic1 can't it be an empty list?

Reply:
if you write `GraphQLNonNull(GraphQLList(GraphQLNonNull(GraphQLFlexTeaserInput)))` you're making sure that the list doesn't have any `null`s in it. It still can be an empty list `[]`. The difference is that in the frontend you don't have to check if the items when you iterate over them because you made sure all items of the list are of type `GraphQLFlexTeaserInput` and nothing else.